lib/cell.rb

```diff
@@ -21,24 +21,22 @@ def fired_upon?
 
   def fire_upon
     @fire = true
-    if empty? == false
-      ship.hit
-    end
+    ship.hit if empty? == false
   end
 
   def render(player_input = false)
     if fired_upon? == false && player_input == false
-      return "."
-    elsif player_input == true && empty? == true && fired_upon? == false
-      return "."
-    elsif fired_upon? == true && empty? == true
-      return "M"
-    elsif fired_upon? == true && empty? == false && ship.sunk? == false
-      return "H"
-    elsif fired_upon? == true && empty? == false && ship.sunk? == true
-      return "X"
-    elsif fired_upon? == false && empty? == false && player_input == true
-      return "S"
+      "."
+    elsif player_input && empty? && fired_upon? == false
+      "."
+    elsif fired_upon? && empty?
```

Comment:
These methods return `true` or `false`. Just calling them will evaluate it as a conditional (no need for the `== true`).